db/migrate/20240928150204_add_guid_to_podcasts.rb

```diff
@@ -0,0 +1,6 @@
+class AddGuidToPodcasts < ActiveRecord::Migration[7.2]
+  def change
+    add_column :podcasts, :guid, :string
```

Comment:
Any reason not to just generate these for all existing Podcasts?

Reply:
Yes, and we could do that at the console at any time, I just didn't force it in this PR